src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig

```diff
@@ -72,14 +72,16 @@
           <strong>{{ 'Validated orders placed:'|trans({}, 'Admin.Orderscustomers.Feature') }}</strong>
         </p>
         <p>
-          <span class="badge rounded badge-dark">{{ orderForViewing.customer.validOrdersPlaced }}</span>
+          <span class="badge rounded badge-dark d-print-none">{{ orderForViewing.customer.validOrdersPlaced }}</span>
+          <span class="badge rounded d-none d-print-inline-block">{{ orderForViewing.customer.validOrdersPlaced }}</span>
```

Comment:
@NeOMakinG I duplicated a lot of items, but each time one has `d-print-none` and the other one has `d-none d-print-...` to have a dedicated display for Print. Is that OK ?

Reply:
Why ? d-print-inline-block is applied only on print screen, it seems useless to do this